generator/src/main/java/com/scottlogic/deg/generator/CommandLine/GenerateCommandLine.java

```diff
@@ -1,14 +1,17 @@
 package com.scottlogic.deg.generator.CommandLine;
 
+import com.google.inject.Guice;
+import com.google.inject.Injector;
 import com.scottlogic.deg.generator.GenerateExecute;
+import com.scottlogic.deg.generator.Guice.BaseModule;
 import com.scottlogic.deg.generator.generation.GenerationConfig;
 import com.scottlogic.deg.generator.generation.GenerationConfigSource;
 import picocli.CommandLine;
 
 import java.io.File;
 import java.nio.file.Path;
 
-public class GenerateCommandLine extends CommandLineBase implements GenerationConfigSource {
```

Comment:
Why are we deleting `CommandLineBase`? Originally, it was added to separate the concerns of having the commandline options and run() within the same class. @sl-slaing what do you think?

Reply:
There used to be 2 command line base inheritors, now there is one. Therefore the abstraction is no longer helpful - we don't need to share that `run()` method with anyone else.

Reply:
But we may need it in the future for the `visualise` command, is that the intention @sl-slaing?

Reply:
Agreed, it shouldn't be removed

Reply:
Have reintroduced the CommandLineBase but I feel this needs rework as part of the IoC phase 2 ticket, #410